python/py-plumed/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+PortGroup           github 1.0
+
+github.setup        plumed plumed2 2.5.0 v
+name                py-plumed
+categories          python
+
+platforms           darwin
+categories-append   science
+license             LGPL-3
+maintainers         {gmail.com:giovanni.bussi @GiovanniBussi} openmaintainer
+
+description         Python wrappers for plumed.
+long_description    ${description} They allow the plumed library to be directly used from python.
+
+homepage            http://www.plumed.org
+
+checksums           rmd160  3873e9d48500bd01353396c0c85723a46b7c0ca9 \
+                    sha256  196aec82920bc6759ac66e32a8673f17e72b6c942bc84aadfd347182c5d9bedf \
+                    size    66557971
+
+python.versions     27 36 37
+
+# python setup is located in python subdir of plumed repository
+pre-configure {
+    worksrcdir ${distname}/python
+}
+
+pre-test {
+    worksrcdir ${distname}/python
+}
```

Comment:
Would it be sufficient just to set worksrcdir globally?

Reply:
You are right, I don't know any more why I did it this way... fixed now!